visivo/models/item.py

```diff
@@ -22,11 +23,14 @@ class Item(BaseModel, ParentModel):
         table: ref(table-name)
       - width: 2
         chart: ref(chart-name)
+      - width: 1
+        chart: ref(selector-name)
```

Comment:
Did you mean to say selector here rather than chart for the key?

Reply:
I did!